plugins/rapid7_insightvm/komand_rapid7_insightvm/util/resource_requests.py

```diff
@@ -103,17 +103,17 @@ def resource_request(
         extras = {"json": payload, "params": parameters.params}
         try:
             response = request_method(url=endpoint, verify=False, **extras)
-        except requests.RequestException as e:
-            assistance = self._REQUEST_EXCEPTIONS.get(type(e), self._UNHANDLED_EXCEPTION)
-            raise PluginException(cause=e, assistance=assistance)
+        except requests.RequestException as error:
+            assistance = self._REQUEST_EXCEPTIONS.get(type(error), self._UNHANDLED_EXCEPTION)
+            raise PluginException(cause=error, assistance=assistance)
 
         resource_request_status_code_check(response.text, response.status_code)
 
         if json_response:
             try:
                 resource = response.json()
-            except json.decoder.JSONDecodeError:
-                raise PluginException(preset=PluginException.Preset.INVALID_JSON)
+            except json.decoder.JSONDecodeError as error:
+                raise PluginException(preset=PluginException.Preset.INVALID_JSON, error=f"Error returned: {error}")
```

Comment:
Shouldn't it be `data` field instead of `error`?
```suggestion
                raise PluginException(preset=PluginException.Preset.INVALID_JSON, data=f"Error returned: {error}")
```

Reply:
You're right thank you 😅